tap_swellrewards/streams.py

```diff
@@ -160,6 +160,8 @@ def sync(self):
                   if transformed_record["last_seen_at"] > most_recent_date:
                     most_recent_date = transformed_record["last_seen_at"]
 
+                singer.bookmarks.write_bookmark(state=self.state, tap_stream_id=self.tap_stream_id, key="last_seen_at", val=most_recent_date)
```

Comment:
@luandy64 I also added this bookmark update, so that if the API craps out without fully finishing, we are still updating the bookmark to the last successful batch of records processed. We've noticed the Swell API is very unreliable, so this should help for when it gives back an error part of the way through.